src/www/views/servers_view/server_list/index.ts

```diff
@@ -38,33 +30,24 @@ export class ServerList extends PolymerElement {
           height: 100%;
         }
 
-        server-card {
+        server-row-card {
```

Comment:
ok so controversially I made two separate card components - the `server-row-card` and the `server-hero-card`. it felt weird to "expand" the server-card, especially since there's an idea of an ["expansion" behavior](https://material.io/components/cards#behavior) in material that the user is supposed to be able to _do_ - in our case these were just two different layouts

there's opportunity here for me to extract the common pieces out and reuse them but idk if you felt that would add indirection

also, when i do the `server-list` i'll use slots to swap between the different layouts.

Reply:
The issue I have is with the inheritance of behavior. The expansion logic in the card is a way to avoid the inheritance of behavior. I have a suggestion to use composition over inheritance in the list item file.